### ISSUE TITLE
Anaconda / pip support?

### ISSUE THREAD

**Issue by thomasaarholt** [delmic/odemis#342]
Just wondering if you have considered/is it possible making odemis available via pip or anaconda?

It would be very nice to have support for odemis via the Jupyter Notebook environment.

**Comment by pieleric** [comment]
Indeed, having it in PIP would be nice for users who want to just use the Odemis functionalities to open and analyse acquisition data.

Currently one limiting factor is that we rely on some "fork" of Pyro4 (although it's not used when just analysing data), and the latest development version of pylibtiff. It should be relatively straightforward to handle the presence of a standard Pyro4. Concerning pylibtiff, it's hopefully just a matter of time before it's available.

**Comment by thomasaarholt** [comment]
As a heads-up, pylibtiff 0.4.1 is now available from [PyPI](https://pypi.python.org/pypi/libtiff).
